pkg/receiver/rawk8seventsreceiver/config.go

```diff
@@ -38,6 +38,8 @@ type Config struct {
 
 	// ConsumeMaxRetries is the maximum number of retries for recoverable pipeline errors
 	ConsumeMaxRetries uint64 `mapstructure:"consume_max_retries"`
+
+	ResourceVersion uint64 `mapstructure:"resource_version"`
```

Comment:
Do we want to make this configurable? Sounds like a footgun.

Reply:
No, we definitely don't. This is a work in progress, far from the final version.